src/libs/TransactionUtils/index.ts

```diff
@@ -1380,7 +1380,7 @@ function buildNewTransactionAfterReviewingDuplicates(reviewDuplicateTransaction:
 
 function buildTransactionsMergeParams(reviewDuplicates: OnyxEntry<ReviewDuplicates>, originalTransaction: Partial<Transaction>): TransactionMergeParams {
     return {
-        amount: -getAmount(originalTransaction as OnyxEntry<Transaction>, false),
+        amount: -getAmount(originalTransaction as OnyxEntry<Transaction>, true),
```

Comment:
I don't quite get this method:
1. Why are there 2 params `isFromExpenseReport` and `isFromTrackedExpense` that do the same thing?
2. What does `isFromTrackedExpense` mean?
3. Why are we passing a hardcoded true/false here, shouldn't it be looking at what report the transaction is in and pass true/false based on that?

Reply:
> Since duplicate resolution happens only in expense reports, not IOU reports.

Not following, is `buildTransactionsMergeParams` only used for expense reports? And if so, where are we validating that we don't pass an IOU?

> This is not the same. Since we can track expense in workspace too and it uses a different convention than expense report, we use isFromTrackedExpense to force a convention irrespective of where the track expense is present.

Sorry, I don't understand what you mean. They both do exactly the same thing in code. If you pass one or the other to true, the exact same logic applies, so to me one of them should be removed.

Reply:
> Nope, one is inverted 

Oh, true! 👨‍🦯